src/core/entities/crux-user.ts

```diff
@@ -91,6 +94,9 @@ export class CruxUser {
     get privateAddresses() {
         return this.cruxUserPrivateAddresses;
     }
+    get blacklist() {
```

Comment:
remove this

Reply:
This is being used in server like - `const isValidPublisher = checkPublisherAccess(cruxUserById!.blacklist, cruxId);`

Reply:
change name to something like `blacklistedCruxIDs` then. there could be all kinds of blacklists

Reply:
No, actually this wouldnt be necessary. blacklist would be accessible via `.config`

Reply:
Done